lib/spack/docs/build_systems/intelpackage.rst

```diff
@@ -0,0 +1,994 @@
+be used unchanged in Spack: Look for the environment variable
+``INTEL_LICENSE_FILE`` or for files
+``/opt/intel/licenses/*.lic`` that contain::
+
+  SERVER  hostname  hostid_or_ANY  portnum
+  USE_SERVER
+
+The relevant tokens, among possibly others, are the ``USE_SERVER`` line,
+intended specifically for clients, and one or more ``SERVER`` lines above it
+which give the network address.
+
+If you cannot find pre-existing ``/opt/intel/licenses/*.lic`` files and the
+``INTEL_LICENSE_FILE`` environment variable is not set (even after you loaded
+any relevant modulefiles), ask your license administrator for the server
+address(es) and place them in a "global" license file within your Spack
+directory tree `as shown below <Spack-managed file_>`_).
+
+
+""""""""""""""""""""""""""""""""""""
+Installing a standalone license file
+""""""""""""""""""""""""""""""""""""
+
+If you purchased a user-specific license, `follow Intel's instructions to
+<https://software.intel.com/en-us/faq/licensing#license-management>`_
+"activate" it for your serial number, then download the resulting license file.
+If needed, `request to have the file re-sent
+<https://software.intel.com/en-us/articles/resend-license-file>`_ to you.
+
+License files are plain text files containing license tokens in FLEXlm format
+and whose name ends in ``.lic``.  Intel installers and compilers look for
+license files in several locations when they run.  Place your license by one of
+the following means, in order of decreasing preference:
+
+* Default directory
+
+  Install your license file in the directory ``/opt/intel/licenses/`` if you
+  have write permission to it. This directory is inspected by all Intel tools
+  and is therefore preferred, as no further configuration will be needed.
+  Create the directory if it does not yet exist.  For the file name, either
+  keep the downloaded name or use another suitably plain yet descriptive
+  name that ends in ``.lic``. Adjust file permissions for access by licensed
+  users.
+
+
+* Directory given in environment variable
+
+  If you cannot use the default directory, but your system already has set the
+  environment variable ``INTEL_LICENSE_FILE`` independent from Spack [fn5]_,
+  then, if you have the necessary write permissions, place your license file in
+  one of the directories mentioned in this environment variable.  Adjust file
+  permissions to match licensed users.
+
+
+  **Recommendation:**
+  If your system has not yet set and used the environment variable
+  ``INTEL_LICENSE_FILE``, you could start using it with the ``spack install``
+  stage of licensed tools and subsequent client packages. You would, however,
+  be in a bind to always set that variable in the same manner, across
+  updates and re-installations, and perhaps accommodate additions to it. As
+  this may be difficult in the long run, we recommend that you do *not* attempt
+  to start using the variable solely for Spack.  Instead, try the next option.
```

Comment:
Right – I can see the confusion.  The preference is conditional, so to speak.  If a site already uses `INTEL_LICENSE_FILE`, then the middle suggestion stands. Otherwise, the decision should fall through to the last one. I'm removing "Instead, try the next option." to remove this meta-flow-control.